doc/whats_new.rst

```diff
@@ -1,3 +1,32 @@
+0.3.1
+=====
+
+Changelog
+---------
+
+API changes summary
+...................
+
+    - The function `nilearn.connectome.sym_to_vec` is is deprecated and will be removed in future
```

Comment:
May be we could say just "`sym_to_vec` from connectome module is deprecated ............." ?

Since you are referring a function as `nilearn.connectome.sym_to_vec` which is not included in doc/modules/reference.rst , you will see documentation build error with CircleCI.